ikarus/python/assembler/flatassembler.hh

```diff
@@ -118,7 +118,10 @@ reduced vector. \n <li> `reducedSize()`: Returns the size of the reduced space.
     cls.def("requirement", &Assembler::requirement);                                                                           \
     cls.def("affordanceCollection", &Assembler::affordanceCollection);                                                         \
     cls.def("dBCOption", &Assembler::dBCOption);                                                                               \
-  }
+    cls.def_property_readonly("size", &Assembler::size);                                                                       \
+    cls.def("__len__", [](Assembler& self) -> int { return self.size(); });                                                    \
```

Comment:
Do we really need to introduce both functions that wrap the same cpp function?

Reply:
I think this is the pythonic way and it is in line with other dune bindings, see.
https://gitlab.dune-project.org/staging/dune-functions/-/blob/master/dune/python/functions/globalbasis.hh?ref_type=heads#L173

Reply:
Yeah I was just thinking about only providing "__len__" and not size because size is non Pythonic. But I if it also inconsistent elsewher in DUNE, I dont care. Why do you use different signatures? `def_property_readonly` for both and both `&Assembler::size` would be more appropriate wouldn it?

Reply:
The `len` function is a build-in function so you can call `len(assembler)` the other is a property e.g. `assembler.size`
I think this is also in line with other python classes, but yeah both should use the member function pointer